README.md

```diff
@@ -231,6 +251,28 @@ For example, if some node is tainted NoSchedule or fails to start a job for some
 
 [![asciicast](https://asciinema.org/a/11wir751y89617oemwnsgli4a.png)](https://asciinema.org/a/11wir751y89617oemwnsgli4a)
 
+### Configuration options
+
+NFD supports a configuration file. The default location is
+`/etc/kubernetes/node-feature-discovery/node-feature-discovery.conf`
+,but, this can be changed by
+specifying the`--config` command line flag. The file is read inside the Docker
+image. Thus, Volumes and VolumeMounts are needed, or, a custom Docker image
```

Comment:
Suggesting user to build a custom image doesn't seem like the best idea to me. I think the better alternative would be to do this with ConfigMap - there could be even an example of it provided and mounted by default in the pods deployed by job/daemonset. ConfigMap are mutable during runtime, so it kinda also solves the issue with the `--options` argument below. If I'd like to rerun NFD with new config, I could just:
1. edit ConfigMap
2. restart process (by killing a pod)

and that's it.
The only downside of it, is that the config is shared between all nodes in the cluster, so if that's the problem we can always fall back to using regulat hostPath volume/volumemounts, which is still way more convenient than rebuilding custom Docker image for each node separately, right?

Reply:
Yeah, I totally agree that this is the preferred and best way. I'll reword the readme to more clearly point this out.

Reply:
Yea, if you would decide to go with the ConfigMap as the main option, you wouldn't even need to add the default config file to the docker image (currently commented out). The less stuff in the docker image the better, right? From the user perspective there's not much difference between configMap and hostPath - it's only where the config file is stored - single place in cluster vs on each node. Both has pros and cons, so it depends on the use-case probably.